docs/source/documentation/patterns/sample-code/_pagination.html

```diff
@@ -1,9 +1,11 @@
-<div class="text-center trailer-1 leader-1">
-  <a href="#" class="btn btn-transparent btn-disabled">Previous</a>
-  <a href="#" class="btn">1</a>
-  <a href="#" class="btn btn-transparent">2</a>
-  <a href="#" class="btn btn-transparent">3</a>
+<div class="pagination">
+  <a href="#" class="pagination-item btn btn-transparent btn-disabled"><svg xmlns="http://www.w3.org/2000/svg" height="16px" width="16px" viewBox="0 0 16 16"><path d="M10.707 12H9.293l-4.5-4.5 4.5-4.5h1.414l-4.5 4.5z"/></svg></a>
```

Comment:
You can use the icon helper here most likely if the icon you're using is part of the calcite-ui set. 

```
{% icon 'x', size=16 %}
```
Documentation here: https://github.com/Esri/calcite-web/blob/2.0/docs/acetate.config.js#L53